### DISCUSSION TITLE
Tab View  slowness

### DISCUSSION THREAD

**Discussion by orenpet** [TomSchimansky/CustomTkinter#1668] [Q&A]
Hi,

I have created 2 tabs one of them is showing a serial port session, and the other is nothing, but moving between them is getting slower as long as the application is running.
why is that?

**Comment by emircetinmemis** [comment]
I've solved it with giving and optimal height & width to tabview. You should give a good width and height that both tabs won't need to make window resize by larging it. Otherwise I couln't solve it by native.
